documentation/docs/06-runtime/02-context.md

```diff
@@ -82,7 +82,7 @@ Context is not inherently reactive. If you need reactive values in context then
 <script>
 	import { setContext } from 'svelte';
```

Comment:
```suggestion
	import { getContext } from 'svelte';
```

Reply:
Oh right, I am so used for those things to me auto-fix in my IDE, haha. Thank you !